.github/ISSUE_TEMPLATE/zgłoszenie-organizacji.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Zgłoszenie organizacji
+about: Szablon do zgłaszania nowych organizacji
+title: "[Organizacja] Nazwa Twojej Organizacji"
+labels: organizacje
+assignees: ''
+
+---
+
+Wypełnij poniższe pola informacjami o swojej organizacji, żeby zgłosić ją do programu. 
+
+```yaml
+nazwa: 
+adres: 
```

Comment:
`adres` czyli adres strony internetowej czy adres fizyczny? Nie jest to jednoznaczne imho. Nie pamiętam, żebyśmy planowali wyświetlanie adresu organizacji

Brakuje numeru KRS, jeżeli wciąż planujemy go pokazywać u siebie

Reply:
Zmienimy to na `www` i dodamy `krs`.